news/2020-05-08-mappers-report-april.md

```diff
@@ -0,0 +1,193 @@
+
+Ever been interested in getting a Beatmap Nominator and deciding what maps get ranked next? Or you just want to help new mappers to get their beatmaps ranked? Then head right over to the [Beatmap Nominator website](https://bn.mappersguild.com/) and apply today! 
+
+### BN Resignations
+
+- **osu!:** [celerih](https://osu.ppy.sh/users/4696296), [Chinochan](https://osu.ppy.sh/users/3626063), [Uta](https://osu.ppy.sh/users/9439796) & [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
+- **osu!catch:** [Daletto](https://osu.ppy.sh/users/7592136) & [Lacrima](https://osu.ppy.sh/users/4915649)
+
+### NAT Additions
+
+Our office cat **[Mao](https://osu.ppy.sh/users/2204515)** has returned! After stepping down from the NAT duties in December 2019, he decided to make his return. Mao will lend the other osu! NATs a helping hand with doing Beatmap Nominator evaluations. Welcome back!
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+The Mappers' Showcase will bring to you Mappers from every game mode which not everyone might know. They are hand-selected and a hidden gem for their game mode. 
+This month will feature osu!taiko and the selectors were [radar](https://osu.ppy.sh/users/7131099) and [Capu](https://osu.ppy.sh/users/2474015).
+
+### Idealism
+
+**[Idealism](https://osu.ppy.sh/users/3869519)** is a osu!taiko mapper (as well as an osu! top player) from Brazil who has been around in the community for a while now, though he definitely deserves more credit for his work in the mapping scene! His maps blend creativity with smooth and interesting patterning which will leave a good taste in the mouths of mappers and players alike.
+
+A map of his that truly shines in many people's eyes is his semi-recently ranked rendition of [Cyte - Command Prompt](https://osu.ppy.sh/beatmapsets/1068145#taiko/2236237). On the top difficulty (mind bogglingly named "C:\Program Files\Difficulties\434f525255505445442046494c45.exe"), Idealism proves that even when mapping genres out of his comfort zone he can still shine. 
+
+Some of his other notable and equally as great maps would be his renditions of [Blue on Blue (TamorlaM Btlg Remix)](https://osu.ppy.sh/beatmapsets/965135#taiko/2020395) and [White Eternity](https://osu.ppy.sh/beatmapsets/992443#taiko/2075446), where he shows off his addiction to eroge music through his solid mapping! We hope to see even more of his maps coming into the ranked section in the rest of 2020, as his potential to shine even brighter clearly shows!
+
+### Yuemiao
+
+Recently making her first appearance in the ranked section with her take on [Mizutama by YUC'e and Nor](https://osu.ppy.sh/beatmapsets/1009290#taiko/2112927),  **[Yuemiao](https://osu.ppy.sh/users/4493348)** shows us that she definitely knows what she's doing.
+
+Executing many different snaps and rhythms perfectly, with clean consistency throughout her difficulties, making this map definitely one of the most interesting ones in the ranked section as of late.
```

Comment:
This sentence is missing a subject (maybe could be merged into the previous one)?